client/public/index.html

```diff
@@ -1,197 +1,78 @@
-            <label id="help" title="Learn more about Try PureScript">Help</label>
-          </a>
-        </li>
-      </ul>
-
-      <div class="mobile-only mobile-banner">
-        Your screen size is too small. Code editing has been disabled.
-      </div>
-
-      <div id="editor_view" data-view-mode="sidebyside">
-        <div id="column1" class="no-mobile">
-
-          <div id="code">
-          </div>
-          <textarea name="code" id="code_textarea" style="display: none;"></textarea>
-        </div>
-
-        <div class="separator">
-        </div>
-
-        <div id="column2_wrapper">
-          <div id="column2"></div>
-          <div id="loading"></div>
-        </div>
-      </div>
-    </div>
-  </div>
-
-  <script type="text/javascript">
-    $.ajaxSetup({
-      dataType: 'text'
-    });
-
-    var editor, cleanupActions = [];
-
-    editor = ace.edit('code');
-
-    editor.renderer.setShowGutter(true);
-    editor.setFontSize(13);
-    editor.setShowPrintMargin(false);
-
-    var session = editor.getSession();
-
-    session.setMode('ace/mode/haskell');
-    session.setOptions({
-      tabSize: 2,
-      useSoftTabs: true
-    });
-
-    function setEditorContent(value) {
-      session.setValue(value);
-    }
-
-    function onEditorChanged(callback, millis) {
-      session.on('change', _.debounce(function() {
-        callback(session.getValue());
-      }, millis));
-    }
+  <head>
+    <title>Try PureScript!</title>
+    <meta content="text/html;charset=utf-8" http-equiv="Content-Type" />
+    <meta content="utf-8" http-equiv="encoding" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+
+    <link rel="icon" type="image/svg+xml" href="/img/favicon-black.svg" />
+    <link rel="alternate icon" href="/img/favicon-black.ico" />
+
+    <script type="text/javascript" src="//cdnjs.cloudflare.com/ajax/libs/jquery/1.12.4/jquery.js" ></script>
+    <script type="text/javascript" src="//cdnjs.cloudflare.com/ajax/libs/ace/1.3.3/ace.js" charset="utf-8" ></script>
+    <script type="text/javascript" src="//cdnjs.cloudflare.com/ajax/libs/ace/1.3.3/mode-haskell.js" ></script>
+    <script type="text/javascript" src="//cdnjs.cloudflare.com/ajax/libs/ace/1.3.3/theme-dawn.js" ></script>
+
+    <link rel="stylesheet" type="text/css" href="//fonts.googleapis.com/css?family=Roboto:300,600" />
+    <link rel="stylesheet" type="text/css" href="//fonts.googleapis.com/css?family=Roboto+Slab:300,600" />
+    <link rel="stylesheet" type="text/css" href="css/index.css" />
+  </head>
+  <body>
+    <script type="text/javascript">
+      $.ajaxSetup({
+        dataType: "text",
+      });
 
-    function cleanUpMarkers() {
-      for (var i = 0; i < cleanupActions.length; i++) {
-        cleanupActions[i]();
+      function teardownIFrame() {
+        var $ctr = $("iframe#output-iframe");
+        $ctr.remove()
```

Comment:
This is necessary because several jQuery functions in PureScript were previously clearing the `column2` container (in order to clear the iframe) before displaying any contents in the column. Since the iframe is still here in jQuery I need a way to imperatively clear the iframe from Halogen.